upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,19 @@
 == Migration Changes
 
+=== Migrating to 8.0.2
+
+==== More Authentication flows changes
+
+REQUIRED and ALTERNATIVE executions not supported at same flow::
+    In previous version, it was possible to have REQUIRED and ALTERNATIVE executions in the same authentication flow at the same level.
+    There were some issues with this approach and we did the refactoring in the Authentication SPI, which means that this is not considered
+    valid anymore. If ALTERNATIVE and REQUIRED executions are configured at the same level, the ALTERNATIVE executions are considered disabled.
```

Comment:
Should the whole second sentence `If ALTERNATIVE .... disabled.` be rendered in bold?